### PR TITLE
Sync the Monitoring module to kube-prometheus

### PR DESCRIPTION
**Pull request by nandajavarma**

### CODE REVIEW

File: katalog/grafana/grafana-config.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: grafana-config
```
Comment (lnovara) [inline katalog/grafana/grafana-config.yml:8]: What is the purpose of this new Secret? I see that it has been added to the `kustomization.yaml` but the Grafana deployment does not make use of it.

Refs from kube-prometheus:
  - https://github.com/prometheus-operator/kube-prometheus/blob/main/manifests/grafana-deployment.yaml#L130
  - https://github.com/prometheus-operator/kube-prometheus/blob/main/manifests/grafana-deployment.yaml#L221

**Review by lnovara — CHANGES_REQUESTED** [review]
Overall LGTM, but while doing some "smoke tests" of the Grafana dashboards I have found the following issues:
- CoreDNS
  - Proxy requests volume panel not showing data
  - Proxy requests 99th percentile duration not showing data
- Kubernetes / Compute Resources / *
  - Double check the CPU and Memory quotas table because I think we are missing some recording rules that are used to build such tables
- Persistent Volumes
  - Both panels not working I think that the underlying metrics changed since we first included the dashboard.

Let me know if I can help you.

File: katalog/grafana/grafana-config.yml (hunk 1 above)
Comment (nandajavarma) [reply katalog/grafana/grafana-config.yml:8]: I have added this secret to the volume mounts

File: katalog/alertmanager-operated/sm.yml [hunk 2]
```diff
@@ -8,11 +8,13 @@ kind: ServiceMonitor
 metadata:
   labels:
     k8s-app: alertmanager
-  name: alertmanager
+  name: alertmanager-main
```
[the remark this answers is not in the document]
Comment (nandajavarma) [reply katalog/alertmanager-operated/sm.yml]: this has been reverted back to `alertmanager`

**Review by lnovara — APPROVED** [review]
LGTM :shipit:

**Pull request merged**